rubocop-bitcrowd.gemspec

```diff
@@ -1,22 +1,41 @@
 Gem::Specification.new do |spec|
-  spec.name          = 'rubocop-bitcrowd'
-  spec.version       = '1.3.0'
-  spec.authors       = ['bitcrowd']
-  spec.email         = ['info@bitcrowd.net']
+  spec.name                 = 'rubocop-bitcrowd'
+  spec.version              = '1.3.0'
+  spec.authors              = ['bitcrowd']
+  spec.email                = ['info@bitcrowd.net']
 
-  spec.summary       = 'The bitcrowd rubocop.yml as a gem.'
-  spec.description   = 'Use this as a quick start to get rubocop with the '\
-                       'settings we use at bitcrowd into your project'
-  spec.homepage      = 'https://github.com/bitcrowd/rubocop-bitcrowd'
-  spec.license       = 'MIT'
+  spec.summary              = 'The bitcrowd rubocop.yml as a gem.'
+  spec.description          = 'Use this as a quick start to get rubocop with the '\
+                              'settings we use at bitcrowd into your project'
+  spec.homepage             = 'https://github.com/bitcrowd/rubocop-bitcrowd'
+  spec.license              = 'MIT'
+  spec.post_install_message = <<~HEREDOC
+
+    PLEASE NOTE: as of version 0.57.0 rubocop allows merging its default
+    AllCops:Include and AllCops:Exclude lists with user defined lists.
+
+    Therefore rubocop-bitcrowd is no longer fully overriding AllCops:Exclude,
+    only adding some extra paths.
```

Comment:
For some reason this section/the "Therefore" here confused me a bit 🤔 Maybe it's just me.

How about something along the lines of…

```
This new version of rubocop-bitcrowd no longer overrides RuboCops default AllCops:Exclude list,
but only contains extra patterns not included in the defaults.

Therefore if you want to keep excluding both the bitcrowd patterns as well as the RuboCop default ones,
add this to your .rubocop.yml
```

?

Reply:
I adapted it somewhat like this.

Reply:
Neat! I like your version even better ❤️